test/parallel/test-config-file.js

```diff
@@ -232,6 +232,32 @@ test('host port flag should be parsed correctly', { skip: !process.features.insp
   strictEqual(result.code, 0);
 });
 
+test('--inspect=true should be parsed correctly', { skip: !process.features.inspector }, async () => {
+  const result = await spawnPromisified(process.execPath, [
+    '--no-warnings',
+    '--expose-internals',
```

Comment:
Why use the internals and not the normal `--inspect`

Reply:
I used the same test structure. internals is used to get the option value

```
require("internal/options").getOptionValue("--inspect")
```

Reply:
Im think you can just do `require('node:inspector').url()` without exposing internals

Reply:
done